AutomatedTesting/Gem/Code/CMakeLists.txt

```diff
@@ -54,3 +54,7 @@ if(PAL_TRAIT_BUILD_SERVER_SUPPORTED)
     set_property(GLOBAL APPEND PROPERTY LY_LAUNCHER_SERVER_PROJECTS AutomatedTesting)
 endif()
 
+# Add project to the list server projects to create the AutomatedTesting.UnifiedLauncher
+if(PAL_TRAIT_BUILD_UNIFIED_SUPPORTED)
+    set_property(GLOBAL APPEND PROPERTY LY_LAUNCHER_UNIFIED_PROJECTS AutomatedTesting)
+endif()
```

Comment:
nit: Linux likes to have newlines at end of files, worth doing a quick pass to add these where missing.

Reply:
We can add a pre-commit hook or update .editorconfig in the repo to automatically insert the final newline.
https://stackoverflow.com/questions/60661412/git-error-if-no-newline-at-end-of-file